### ISSUE TITLE
Lua errrors on binding

### ISSUE THREAD

**Issue by seathasky** [AliekberFFXI/xivcrossbar#11]
![image](https://user-images.githubusercontent.com/17598024/179510734-da341dd9-6692-48f4-ab43-9790c92d6a33.png)
Sometimes when making a binding console will shoot LUA errors and crossbar becomes unusable until restart

**Comment by AliekberFFXI** [comment]
@seathasky Can you describe what steps you took that led to the error? Also, did a whole bunch of errors appear at once when you reached the "Bind Protect?" dialog? Or did they happen 1 by 1 as you pressed buttons?

**Comment by seathasky** [comment]
Clicked the PS4 controller Share button to bind a key, am asked what type of binding I want to do, choose protect for WHM, asked what bind I want, choose my bind. Get LUA error, proceed to try a few different bind combos and LUA errors continue.

**Comment by AliekberFFXI** [comment]
@seathasky I'm so far unable to reproduce this. Are you on the latest version of the addon?

**Comment by seathasky** [comment]
> @seathasky I'm so far unable to reproduce this. Are you on the latest version of the addon?

Yep, on latest. Have also replicated the issue on a fresh install on a different setup (my laptop)

**Comment by seathasky** [comment]
This issue was tested on Retail and Eden (Pserver) You can bypass it by pressing the L2 and R2  paddles until it removes the binding on screen
